webplugin/js/app/mck-sidebox-1.0.js

```diff
@@ -10161,7 +10168,18 @@ var userOverride = {
                     contact.contactId,
                     isGroupTab
                 );
-                var imgsrctag = _this.getContactImageLink(contact, displayName);
+                var imgsrctag = '';
+                var isConversationInWaitingQueue = false
+                if (contact && contact.metadata && contact.metadata.CONVERSATION_STATUS == 7 ){
```

Comment:
do we have any constant for `7`?

Reply:
we have it in Kommunicate.conversationHelper.status
I was searching it in KommunicateConstants